CHANGELOG

```diff
@@ -0,0 +1,22 @@
+## What's Changed
+* Increase support for unions in incremental smt2 decision procedure by @thomasspriggs in https://github.com/diffblue/cbmc/pull/7926
+* Add ASV book CBMC examples by @peterschrammel in https://github.com/diffblue/cbmc/pull/7921
+* Add models for C library: getpwnam, getpwuid, and getopt by @tautschnig in https://github.com/diffblue/cbmc/pull/7919 and https://github.com/diffblue/cbmc/pull/7916
+* Shadow memory addresses now also support --pointer-check argument by @esteffin in https://github.com/diffblue/cbmc/pull/7936
+* [DOCS] Add documentation on CBMC Shadow Memory intrinsics by @NlightNFotis in https://github.com/diffblue/cbmc/pull/7913
+* [DOCS] Improve doxygen for shadow_memory_util by @NlightNFotis in https://github.com/diffblue/cbmc/pull/7930
+* [SYNTHESIZER] Allow goto-synthesizer accept all CBMC options  by @qinheping in https://github.com/diffblue/cbmc/pull/7900
+
+## Bug Fixes
+* Fix problem on array size L2 renaming by @esteffin in https://github.com/diffblue/cbmc/pull/7877
+* Fix shadow memory missing aggregation on non-compound bitvector types by @esteffin in https://github.com/diffblue/cbmc/pull/7935
+
+## Internal Improvements
```

Comment:
I suggest not even including this section, though perhaps those could be listed in the PR description for reviewers to potentially override whoever is putting together the PR. My personal take is that the changelog should strongly focus on observable changes, where "observable" are all those changes that impact users in the first instance and developers if there are substantial API changes.